engine/orchestration/src/main/java/com/cloud/vm/VirtualMachineManagerImpl.java

```diff
@@ -1351,6 +1360,12 @@ public void orchestrateStart(final String vmUuid, final Map<VirtualMachineProfil
                 if (VirtualMachine.Type.User.equals(vm.type) && ResourceCountRunningVMsonly.value()) {
                     resourceCountDecrement(owner.getAccountId(),new Long(offering.getCpu()), new Long(offering.getRamSize()));
                 }
+                // Decrement VR resource count if the VR can't be started
```

Comment:
As the comment was intended to be a literal description of the following line, it is not necessary.